### PR TITLE
Add health check for tier4 tests before they are run

### PR DESCRIPTION
**Pull request by vasukulkarni**
Fixes: https://github.com/red-hat-storage/ocs-ci/issues/1315

Signed-off-by: Vasu Kulkarni <vasu@redhat.com>

### CODE REVIEW

File: tests/conftest.py [hunk 1]
```diff
@@ -798,6 +799,22 @@ def polarion_testsuite_properties(record_testsuite_property, pytestconfig):
...
+    tier4_marks = [m.mark for m in [tier4a, tier4b, tier4c]]
+    for mark in node.iter_markers():
+        if mark in tier4_marks:
+            log.info("Checking for Ceph Health OK ")
+            status = ceph_health_check()
+            if status:
+                log.info("Health check passed")
+                return
+            else:
+                pytest.skip("Ceph Health check failed")
```
Comment (vavuthu) [inline tests/conftest.py]: why would we need to skip the test if ceph health is not OK. I believe we need to raise assert here.

File: ocs_ci/utility/utils.py [hunk 2]
```diff
@@ -1186,7 +1186,14 @@ def get_az_count():
 
 
 @retry((CephHealthException, CommandFailed), tries=20, delay=30, backoff=1)
-def ceph_health_check(namespace=None):
+def ceph_health_check(namespace=None, tries=20, delay=30, backoff=1):
```
Comment (clacroix12) [inline ocs_ci/utility/utils.py]: The arguments `tries`, `delay`, and `backoff` aren't used, can we remove them? We will also want to add `namespace` to the docstring.

Comment (vasukulkarni) [reply ocs_ci/utility/utils.py]: I will fix both

**Comment by vasukulkarni** [comment]
Working as expected: 2 failures (+2) , 48 skipped (+48)

**Pull request merged**